psique/src/App/test/wais/subtest/aritmetica.js

```diff
@@ -1,8 +1,385 @@
+      }
+
+      if(badAnswerCount > 0){        
+        badAnswerCount = 0;
+      }
+      
+      setResults(update(results,{
+        [numberItem]: {
+          $set: 1
+        }}))
+      
+      changeStimuli(1);
+      
+    }else{
+      setResults(update(results,{
+        [numberItem]: {
+          $set: 0
+        }}))
+
+      //if(numberItem !== 0){
+        badAnswerCount += 1;
+      //}
+
+      if(countRe>0){
+        countRe=0;
+      }
+
+      changeStimuli(0);
+    }    
+  }
+
+  function content(){
+    switch (state) {
+      case 'instruccion':
+       return (
+        <div>
+          <h1>Aritmetica</h1>
+          <b>Intrucciones generales:</b>
+          <p>A continuación se mostrará una serie de imágenes (reactivo 1/retorno) o preguntas (reactivo 6)</p>
+          <p>según sea el caso, las cuales, en el caso de las imágenes deben ser mostradas al paciente para que</p>
+          <p>las visualice, y en el caso de las preguntas deben ser mencionadas al paciente</p>
+          <br/>
+          <b>Intrucciones para registrar la respuesta de paciente:</b>
+          <br/>
+          <br/>
+          <li>La respuesta dada por el paciente debe ser registrada en la casilla de respuesta</li>
+          <p>El sistema calificara automaticamente la respuesta</p>
+          <br/>
+          <CustomButton
+            msj="Iniciar subprueba"
+            callback={next}
+          ></CustomButton>
+        </div>
+       );
+
+      case "seleccion":
+       return(
+        <div>
+          <h1>Aritmética</h1>
+          <p>En que estimulo desea iniciar la prueba? </p>
+          <p>Pacientes con sospechas de discapacidad intelectual:</p>
+          <CustomButton msj="Reactivo 1"
+          callback={()=>imagenInit(1)}></CustomButton>          
+          <br/><br/>
+          <p>Pacientes de edad 16-89:</p>
+          <CustomButton msj="Reactivo 6"
+          callback={()=>imagenInit(6)}></CustomButton>
+        </div>
+       );
+
+       case "ejemplo":
+       return(
+         <div>
+            <h1>Reactivo Ejemplo. {consigna[numberItem]}</h1>
+            <br></br>        
+            <TextField
+              label = "Respuesta dada"
+              value={givenAnswer}
+              variant="outlined"
+              onChange={(x)=>{setGivenAnswer(x.target.value)}}
+            />
+            <br/><br/>
+            <CustomButton
+              msj="Siguiente"
+              callback={next}
+            ></CustomButton>
+          </div>
+       );
+
+       case "aplicacionImg":
+       return(
+         <div>
+            <h1>Reactivo {numberItem}. {consigna[numberItem]}</h1>
+            <img 
+              className={classes.img}
+              alt={"Estimulo "+stimuliSource[numberItem]}
+              src={require("../../../assets/estimulos/aritmetica/"+stimuliSource[numberItem-1]+".jpg")}
+            />
+            <br></br><br></br><br></br>
+            <TextField
+              label = "Respuesta dada"
```

Comment:
There are some questions that must be answered with words too (such as question 15 ), so you can add the text  outside the textfield to avoid enter text in the field.